docs/xbcloud-binary-fifo-datasink.md

```diff
@@ -0,0 +1,64 @@
+# FIFO data sink
+
+The feature is in [tech preview](glossary.md#tech-preview).
+
+[Percona XtraBackup 8.0.33-28](release-notes/8.0/8.0.33-28.0.md) implements a new data sink that uses the first in, first out (FIF0) method. With the `FIFO` data sink feature, you can benefit from faster backups by streaming data in parallel from XtraBackup to `xbcloud`.
```

Comment:
Same here (as mentioned in the release notes), let's mention about the intended audience first.

ie. When should users use this feature?

Reply:
Agree. We should explicitly mention that this requires a 10gbps link between source and destination (server or object storage)